snap/snapcraft.yaml

```diff
@@ -38,6 +38,7 @@ parts:
         sed -rin 's/.*git.*diff.*--quiet.*//' $SNAPCRAFT_PART_SRC/dvc/version.py
     override-build: |
         snapcraftctl build
+        sed -ri 's/^(ENABLE_USER_SITE = )None$/\1False/' $SNAPCRAFT_PART_INSTALL/usr/lib/python*/site.py
```

Comment:
Let's put a comment about what is going on here, please.

Reply:
not actually sure the issue is fixed (https://github.com/iterative/dvc/issues/3514#issuecomment-634281454) - will do some more debugging

Reply:
related: https://github.com/iterative/dvc/tree/snap-site-0.x

Reply:
comment added